engine/table/src/main/java/io/deephaven/engine/table/impl/by/PartitionByChunkedOperator.java

```diff
@@ -820,9 +904,13 @@ private void propagateUpdatesToModifiedDestinations(@NotNull final RowSequence m
                 final long firstBackingChunkDestination =
                         tables.resetWritableChunkToBackingStore(tablesResettableChunk, firstSliceDestination);
                 // The (valid) assumption is that the other write-through resets will address the same range.
-                addedRowSets.resetWritableChunkToBackingStore(addedRowSetsResettableChunk, firstSliceDestination);
-                removedRowSets.resetWritableChunkToBackingStore(removedRowSetsResettableChunk, firstSliceDestination);
-                modifiedRowSets.resetWritableChunkToBackingStore(modifiedRowSetsResettableChunk, firstSliceDestination);
+                addedBuilders.resetWritableChunkToBackingStore(addedBuildersResettableChunk,
+                        firstSliceDestination);
+
```

Comment:
Weird extra new blank line is distracting, here and in subsequent blocks. Is there a reason to separate added from the rest?